README.md

```diff
@@ -37,6 +37,12 @@ Check out our [tutorial on configuring and building these samples on Ubuntu](htt
 	- **Dependencies:**
 		- [OpenCV](https://opencv.org/) version 4.0.1-dev or newer
 
+- [**HandEyeCalibration**](https://github.com/zivid/cpp-extra-samples/tree/master/HandEyeCalibration)
+	- [**UtilizeEyeInHandCalibration**](https://github.com/zivid/cpp-extra-samples/blob/master/HandEyeCalibration/UtilizeEyeInHandCalibration/UtilizeEyeInHandCalibration.cpp) - Transform a 3D point from camera frame to robot base frame using hand-eye calibration matrix.
+	- **Dependencies:**
+		- [OpenCV](https://opencv.org/) version 4.0.1-dev or newer
+		- [Eigen](http://eigen.tuxfamily.org/) version 3.3.90 or newer
```

Comment:
Can you check if this will build with the latest stable Eigen release - 3.3.7. We use 3.3.90 because it is the only one that works for Downsampling project.

Reply:
works with 3.37 as well.

Reply:
Let's use 3.37 then for this.

Reply:
done